go.mod

```diff
@@ -520,6 +543,7 @@ replace (
 	gopkg.in/inf.v0 => gopkg.in/inf.v0 v0.9.1
 	gopkg.in/mcuadros/go-syslog.v2 => gopkg.in/mcuadros/go-syslog.v2 v2.2.1
 	gopkg.in/natefinch/lumberjack.v2 => gopkg.in/natefinch/lumberjack.v2 v2.0.0
+	gopkg.in/olivere/elastic.v2 => gopkg.in/olivere/elastic.v2 v2.0.12
```

Comment:
this is only used by cadvisor elasticsearch storage (which we don't use). we should find a way to isolate storage and container integrations in cadvisor so we don't pull this into our dependency tree if we don't need to